<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>raku</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 13pt &quot;Source Code Pro&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_Toolbox">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="toolButton_Pencil">
             <property name="text">
              <string>.</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_Dropper">
             <property name="text">
              <string>.</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_Bucket">
             <property name="text">
              <string>.</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_Crop">
             <property name="text">
              <string>.</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_Shift">
             <property name="text">
              <string>.</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea_Image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>514</width>
             <height>305</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_Palette">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>128</height>
           </size>
          </property>
          <property name="title">
           <string>Palette</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="font">
           <font>
            <family>Source Code Pro</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_Preview">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>143</width>
             <height>153</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Preview</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_Layers">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>143</width>
             <height>153</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Layers</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_Brushes">
           <attribute name="label">
            <string>Brushes</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_SaveAll"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuGrid">
     <property name="title">
      <string>Grid</string>
     </property>
     <addaction name="actionShow_Grid"/>
     <addaction name="actionConfigure"/>
    </widget>
    <addaction name="menuGrid"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuColors">
    <property name="title">
     <string>Colors</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSelect"/>
   <addaction name="menuImage"/>
   <addaction name="menuColors"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_SaveAll">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_ToggleGrid">
   <property name="text">
    <string>Toggle Grid</string>
   </property>
  </action>
  <action name="action_ConfigureGrid">
   <property name="text">
    <string>Configure Grid ...</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="action_ConfigGrid">
   <property name="text">
    <string>cfuuu</string>
   </property>
  </action>
  <action name="actionConfigure_Grid">
   <property name="text">
    <string>Configure Grid</string>
   </property>
  </action>
  <action name="actioncfgrd">
   <property name="text">
    <string>cfgrd</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
